acceptance-tests/src/test/resources/features/UKDrivingLicence/DVA/DVADrivingLicence.feature

```diff
@@ -180,6 +180,42 @@ Feature: DVA Driving Licence Test
     And JSON response should contain error description Authorization permission denied and status code as 302
     And The test is complete and I close the driver
 
+  @DrivingLicenceTest @build @staging @integration
+  Scenario:User Selects DVA and landed in DVA page
+    Given I click on DVA radio button and Continue
+    Then I should be on the page DVA Enter your details exactly as they appear on your UK driving licence - Prove your identity - GOV.UK
+    And The test is complete and I close the driver
+
+  @DVADrivingLicence_test @build
+  Scenario Outline: DVAError tab title validation
+    Given I click on DVA radio button and Continue
+    When I should be on the page DVA Enter your details exactly as they appear on your UK driving licence - Prove your identity - GOV.UK
+    And I check the page title is Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    Then User enters DVA data as a <DVADrivingLicenceSubject>
+    And User clicks on continue
+    Then I check the page title is Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    And The test is complete and I close the driver
+    Examples:
```

Comment:
Should this not really live in the FE validation/validation-regression tests?

Reply:
Yeah but I would take that on another ticket. This is a known issue